### ISSUE TITLE
Check that rta env does or does not need source environment in the merging stage

### ISSUE THREAD

**Issue by garciagenrique** [cta-observatory/lstmcpipe#76]
Same as #75

**Comment by garciagenrique** [comment]
Not only the RTA env needs this. Indeed, the `r0_to_dl1`, `merge` and `training` stages need the same env, otherwise the hdf5 lib does not like it. 

It will pop an error of the kind;

```
(rta) enrique.garcia@cp01 DL1 $ lstchain_merge_hdf5_files -d /fefs/aswg/workspace/sami.caroff/RealDataHDF5_rta_runs/DL1/subruns_3947/ -o ./dl1_LST-1.Run03947.h5 --no-image True --smart False
  0%|                                                                                                                                                                              | 0/64 [00:00<?, ?it/s]Traceback (most recent call last):
  File "/fefs/aswg/workspace/enrique.garcia/miniconda3/envs/rta/bin/lstchain_merge_hdf5_files", line 8, in <module>
    sys.exit(main())
  File "/fefs/aswg/workspace/enrique.garcia/miniconda3/envs/rta/lib/python3.7/site-packages/lstchain/scripts/lstchain_merge_hdf5_files.py", line 88, in main
    auto_merge_h5files(file_list, args.outfile, nodes_keys=keys)
  File "/fefs/aswg/workspace/enrique.garcia/miniconda3/envs/rta/lib/python3.7/site-packages/lstchain/io/io.py", line 201, in auto_merge_h5files
    obj=f1.root[k].read())
  File "/fefs/aswg/workspace/enrique.garcia/miniconda3/envs/rta/lib/python3.7/site-packages/tables/table.py", line 1929, in read
    arr = self._read(start, stop, step, field, out)
  File "/fefs/aswg/workspace/enrique.garcia/miniconda3/envs/rta/lib/python3.7/site-packages/tables/table.py", line 1843, in _read
    self._read_records(start, stop - start, result)
  File "tables/tableextension.pyx", line 580, in tables.tableextension.Table._read_records
tables.exceptions.HDF5ExtError: HDF5 error back trace
  File "H5Dio.c", line 199, in H5Dread
    can't read data
  File "H5Dio.c", line 603, in H5D__read
    can't read data
  File "H5Dchunk.c", line 2293, in H5D__chunk_read
    unable to read raw data chunk
  File "H5Dchunk.c", line 3658, in H5D__chunk_lock
    data pipeline read failed
  File "H5Z.c", line 1326, in H5Z_pipeline
    filter returned failure during read
  File "hdf5-blosc/src/blosc_filter.c", line 254, in blosc_filter
    Blosc decompression error
End of HDF5 error back trace
Problems reading records.
  0%|                
```

**Comment by garciagenrique** [comment]
Closed with #75 for both lstchain and hiperta workflow kinds
